### PR TITLE
Updated README.md and CONTRIBUTORS.md

### PR DESCRIPTION
**Pull request by vchrombie**
Thank you for your contribution. Please provide the details requested below.

## ISSUE NUMBER

#67 #21 

## SHORT DESCRIPTION

- Added few badges in the README

- Fixed few Broken Links

- Made a seperate table for Kharagpur Winter of Code

## TESTING

You can see the README.md and CONTRIBUTORS.md

### CODE REVIEW

**Comment by vchrombie** [comment]
Can you review this PR and tell me if you want any changes.

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 [![Build Status](https://travis-ci.org/prateekiiest/Code-Sleep-Python.svg?branch=master)](https://travis-ci.org/prateekiiest/Code-Sleep-Python)
 
 
+[![chat on Slack](https://img.shields.io/badge/chat%20on%20-Slack-blue.svg)](https://join.slack.com/t/code-sleep-python/shared_invite/enQtMjc1NDk2ODk4Mjc2LTI3NmM1OWQ5YjcwYzlkN2YxNmZhNWI2NzE0M2Q4MTNiMzM0Y2RmMzg4YWVkODEwNWI3NTUyN2E0OGY5OWZmZTQ)
+
+[![made with &hearts in Python](https://img.shields.io/badge/made%20with%20%E2%9D%A4%20in-Python-red.svg)](http://shields.io/#your-badge)
+
 # Join Slack through this [link](https://join.slack.com/t/code-sleep-python/shared_invite/enQtMjc1NDk2ODk4Mjc2LTI3NmM1OWQ5YjcwYzlkN2YxNmZhNWI2NzE0M2Q4MTNiMzM0Y2RmMzg4YWVkODEwNWI3NTUyN2E0OGY5OWZmZTQ)
```
Comment (prateekiiest) [inline README.md]: you can remove this link now

Comment (vchrombie) [reply README.md]: Okay!

**Comment by vchrombie** [comment]
Is it okay now?

**Comment by prateekiiest** [comment]
Looks good 🎆 

Thanks for the contribution :1st_place_medal:

**Pull request merged**